DemoApp/DemoApp/TMInputPage.xaml

```diff
@@ -54,13 +46,19 @@
                     TitleText="Right Icon" 
                     Placeholder="Text Input" 
                     RightIconSource="password_eye.png" />
-
+                
                 <Modus:TMInput 
                     TitleText="Both Icon" 
                     LeftIconSource="search_icon.png" 
                     Placeholder="Text Input" 
                     RightIconSource="password_eye.png" />
                 
+                <Modus:TMInput 
```

Comment:
We moved this to the top as we cannot see the validation fields as the keyboard hides them. Please confirm if you can see them

Reply:
The changes for input are pushed to develop from the fix/sprint1b with a PR open now but not pulled to the current toast branch .